test/insync.js

```diff
@@ -4771,5 +4735,22 @@ describe('Insync', function () {
                 done();
             });
         });
+
+        describe('_hash()', function () {
+
+            it('returns an object with a null prototype', function (done) {
+
+                var result = Util.hash();
+                expect(result.hasOwnKey).to.be.undefined();
```

Comment:
`hasOwnKey` should probably be `hasOwnProperty`. Maybe add a check for `result.constructor` too.